chapters/my_first_jina_app/index.md

```diff
@@ -0,0 +1,389 @@
+# Build your first Jina app
+
+## 👋 Introduction
+
+This tutorial guides you through building your own neural search app using the [Jina framework](https://github.com/jina-ai/jina/). 
+
+![](./images/jinabox-wikipedia.gif)
+
+Our example program will be a simple neural search engine for text. It will take a user's input, and return a list of sentences from Wikipedia that match most closely.
+
+The end result will be close to [wikipedia sentence search](https://github.com/jina-ai/examples/tree/master/wikipedia-sentences).
+
+## 🗝️ Key concepts
```

Comment:
What are key concepts? Things you should have looked at before, or things you learn by reading this?